content/ko/training/_index.html

```diff
@@ -0,0 +1,108 @@
+---
+title: 훈련하기
+bigheader: 쿠버네티스 교육과 인증
+abstract: 교육 프로그램, 인증 그리고 파트너
+layout: basic
+cid: training
+class: training
+---
+
+<div class="padded blue-bg">
+  <div class="main-section two-thirds-centered white-text">
+    <center>
+      <h2>당신의 클라우드 네이티브 커리어를 구축하세요</h2>
+      <p>쿠버네티스는 클라우드 네이티브 무브먼트의 핵심입니다. 리눅스 재단과 교육 파트너의 교육 그리고 인증을 통해 당신의 커리어에 투자하고, 쿠버네티스를 배우며 클라우드 네이티브 프로젝트를 성공적으로 수행할 수 있습니다.</p>
+    </center>
+  </div>
+</div>
+
+<section>
+  <div class="main-section padded">
+    <center>
+      <h2>edX에서 무료 강좌 수강하기</h2>
+    </center>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>쿠버네티스 소개 <br> &nbsp;</b>
+        </h5>
+        <p>쿠버네티스를 배우고 싶습니까? 컨테이너화된 애플리케이션의 관리를 위한 강력한 시스템에 대한 심도있는 입문서를 얻으십시오.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button">강좌로 이동하기</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>클라우드 인프라 기술 소개</b>
+        </h5>
+        <p>오픈 소스의 리더인 리눅스 재단으로부터 직접 클라우드 기술을 구축하고 관리하는 기본사항을 배웁니다.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button">강좌로 이동하기</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>리눅스 소개</b>
+        </h5>
+        <p>리눅스를 배운적이 없습니까? 새롭게 되고 싶으신가요? 주요 리눅스 배포판에서 그래픽 인터페이스와 명령줄을 모두 사용해서 리눅스에 대한 우수한 실무 지식을 발전시킬 수 있습니다.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-linux" target="_blank" class="button">강좌로 이동하기</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div class="padded lighter-gray-bg">
+  <div class="main-section two-thirds-centered">
+    <center>
+      <h2>리눅스 재단과 함께 배우기</h2>
+      <p>리눅스 재단은 쿠버네티스 애플리케이션 개발과 운영 수명주기의 모든 측면을 강사 주도와 자가 학습 과정을 제공합니다.</p>
+      <br><br>
+      <a href="https://training.linuxfoundation.org/training/course-catalog/?_sft_technology=kubernetes" target="_blank" class="button">강좌 보기</a>
+    </center>
+  </div>
+</div>
+
+<section>
+  <div class="main-section padded">
+    <center>
+        <h2>쿠버네티스 인증 획득하기</h2>
+    </center>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>쿠버네티스 애플리케이션 개발자 인증 (CKAD)</b>
+        </h5>
+        <p>쿠버네티스 애플리케이션 개발자 인증(The Certified Kubernetes Application Developer (CKAD)) 시험은 사용자가 쿠버네티스의 클라우드 네이티브 애플리케이션을 디자인, 구축, 구성과 노출할 수 있음을 인증합니다.</p>
```

Comment:
```suggestion
        <p>공인 쿠버네티스 애플리케이션 개발자 시험은 사용자가 쿠버네티스의 클라우드 네이티브 애플리케이션을 디자인, 구축, 구성과 노출을 할 수 있음을 인증합니다.</p>
```

1. 나열된 단어의 형태가 모두 명사여야 자연스럽습니다.